node/core/approval-voting/src/criteria.rs

```diff
@@ -549,6 +822,22 @@ fn is_in_backing_group(
 	validator_groups.get(group).map_or(false, |g| g.contains(&validator))
 }
 
+/// Migration helpers.
+impl From<crate::approval_db::v1::OurAssignment> for OurAssignment {
+	fn from(value: crate::approval_db::v1::OurAssignment) -> Self {
+		Self {
+			cert: value.cert.into(),
+			tranche: value.tranche,
+			validator_index: value.validator_index,
+			// Whether the assignment has been triggered already.
+			triggered: value.triggered,
+			// This is a dummy value, assignment bitfield will be set later.
+			// The migration sanity check will test for 1 single bit being set here.
+			assignment_bitfield: dummy_assignment_bitfield(),
```

Comment:
Yes, but since this is temporary code as it is for all migrations, I wouldn't invest much in it. It would also complicate the code, as we use OurAssignment later on and will have to actually handle the variants.

Reply:
removed `assignment_bitfield` completely :)